src/applications/personalization/profile/components/ProfileHeader.jsx

```diff
@@ -124,6 +138,26 @@ const ProfileHeader = ({
                 {getServiceBranchDisplayName(latestBranchOfService)}
               </dd>
             )}
+            {showUpdatedHeader &&
```

Comment:
We probably don't _need_ this `showUpdatedHeader` check, but an additional level of protection is fine.

Reply:
I added this with the initial intention to also duplicate this updated header logic onto the profile, which would require this check since we still want to render the current header if the FF is turned off. Since i reverted my logic around also showing this updated banner in the profile, this check appears redundant but I'll leave it since we will need it once the profile can also show this updated header.